File: task2.js
async function generateCard() {

    const name = document.getElementById("name").value;
    const bio = document.getElementById("bio").value;
    const image = document.getElementById("image").value;

    const response = await fetch("/profile", {
        method: "POST",
        headers: {
            "Content-Type": "application/json"
        },
        body: JSON.stringify({
            name,
            bio,
            image
        })
    });

    const data = await response.json();

    document.getElementById("profileCard").innerHTML = `
        <div class="card">
            <img src="${data.image}" alt="Profile Image">
            <h2>${data.name}</h2>
            <p>${data.bio}</p>
        </div>
    `;
}

File: task2.py
from flask import Flask, render_template, request, jsonify

app = Flask(__name__)

@app.route("/")
def home():
    return render_template("index.html")

@app.route("/profile", methods=["POST"])
def profile():
    data = request.get_json()

    return jsonify({
        "name": data["name"],
        "bio": data["bio"],
        "image": data["image"]
    })

if __name__ == "__main__":
    app.run(debug=True)